db/migrations/20190911160416_drop_title_from_movies.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.schema.table('movies', (table) => {
```

Comment:
Could you try using `async` and `await` here? It might be good practice! It not only helps to avoid nested promises or callback hell, it's also cleaner to read.

e.g. `await Knex.schema.table(...`